src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -1358,8 +1392,9 @@ works for every format with 2d and 2d-array textures.
       textureDataLayout: { offset, bytesPerRow, rowsPerImage },
       copySize,
       dataSize,
-      textureSize: [4 * info.blockWidth, 4 * info.blockHeight, copyDepth],
+      textureSize: [4 * info.blockWidth, textureHeight, copyDepth],
```

Comment:
maybe should pull the width out above as well so all the size computations are colocated.

Reply:
This is a bit hard because `uploadTextureAndVerifyCopy` expects a `[number, number, number]` and that's only possible if I use `as const`, which prevents modifying `textureSize`. I moved `textureSize` right after the block  that computes `textureHeight` though.